frontend/packages/console-app/src/plugin.tsx

```diff
@@ -81,7 +82,8 @@ const plugin: Plugin<ConsumedExtensions> = [
     type: 'Perspective',
     properties: {
       id: 'admin',
-      name: 'Administrator',
+      // t('console-app~Administrator')
```

Comment:
hi @rebeccaalpert do we need this comment?

Reply:
All set @jcaianirh.